### DISCUSSION TITLE
Support for Quill 2.0.0

### DISCUSSION THREAD

**Discussion by jbkly** [primefaces/community#1182] [PrimeReact] [answered]
The PrimeReact `Editor` depends on Quill v1.3.7, which was [last updated in 2019](https://github.com/quilljs/quill/releases/tag/v1.3.7), relies on [deprecated browser events](https://github.com/quilljs/quill/issues/3806), and exposes a [cross-site scripting vulnerability](https://github.com/quilljs/quill/issues/3364).

The past 2 months have seen active development on Quill 2.0.0. There is now a [release candidate](https://github.com/quilljs/quill/releases), [updated documentation](https://v2.quilljs.com/) and [migration guide to v2](https://v2.quilljs.com/guides/upgrading-to-2-0), and a [stable 2.0.0 release planned for March 1](https://github.com/quilljs/quill/issues/3806#issuecomment-1933250524).

Are there plans to update PrimeReact's dependency on Quill to the new version?

**Comment by melloware** [comment, marked as answer]
@jbkly Yes.  I opened a Ticket on PrimeFaces for Quill 2.0.0 to remind myself I think you will see when Quill 2.0.0 goes final that PrimeReact will update to it.

**Comment by jbkly** [comment]
Thanks, I see it and will track it there: https://github.com/primefaces/primereact/issues/5960

**Reply by melloware** [reply]
PR submitted: https://github.com/primefaces/primereact/pull/6429
